mountpoint-s3/src/inode.rs

```diff
@@ -2797,4 +2809,53 @@ mod tests {
         assert_eq!(file_inodestat.ctime, ts);
         assert_eq!(file_inodestat.mtime, ts);
     }
+
+    #[cfg(feature = "shuttle")]
+    mod shuttle_tests {
```

Comment:
Does it make sense to add a test for a file X that the kernel calls `forget` on but simultaneously we get a `create` request? I'm thinking we are in the middle of processing a request for `forget` while in parallel we get a request to create a file of the same name and same directory.

Reply:
Although it will probably be different inodes, so maybe it does not make sense.

Reply:
Yeah, it will be different inodes. I will leave the test as it is for now.